certstream/core.py

```diff
@@ -16,13 +16,13 @@ class CertStreamClient(WebSocketApp):
     CERTSTREAM_URL = 'wss://certstream.calidog.io'
```

Comment:
Awesome, thanks for the PR. Mind also removing this as part of your change-set? Not sure it's used anywhere else!